qcodes/instrument_drivers/rigol/DG4000.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, name, address, reset=False, **kwargs):
             ramp_freq = [5e6, 4e6, 3e6, 1e6][i]
             pulse_freq = [50e6, 40e6, 25e6, 15e6][i]
             harmonic_freq = [100e6, 80e6, 50e6, 30e6][i]
-            arb_freq = [50e6, 40e6, 25e6, 15e6][i]
+            self.arb_freq = [50e6, 40e6, 25e6, 15e6][i]
```

Comment:
`self.` should be added to all of these `*_freq` then, because they are all (?) used below in `_source_apply` calls.